examples/async/package.json

```diff
@@ -7,7 +7,7 @@
     "test": "cross-env NODE_ENV=test babel-node test/sagas.js | tap-spec"
   },
   "dependencies": {
-    "redux-saga": "^1.0.0-beta.0"
+    "redux-saga": "file:../../packages/core"
```

Comment:
actually I would prefer not relaying on those packages being in a monorepo and stuff, with "real" dependencies we can in example easily upload those examples to things like codesandbox.io etc and it should download its copies of those deps

Reply:
ok. I'll revert it